Symfony/CS/Test/IntegrationCaseFactory.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Test;
+
+use Symfony\CS\Fixer;
+use Symfony\CS\FixerInterface;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+final class IntegrationCaseFactory
+{
+    private static $builtInFixers;
+
+    /**
+     * @param string $fileName
+     * @param string $content
+     *
+     * @return IntegrationCase
+     */
+    public function create($fileName, $content)
+    {
+        try {
+            if (!preg_match('/--TEST--\n(?<title>.*?)\s--CONFIG--\n(?<config>.*?)(\s--SETTINGS--\n(?<settings>.*?))?(\s--REQUIREMENTS--\n(?<requirements>.*?))?\s--EXPECT--\n(?<expect>.*?\n*)(?:\n--INPUT--\s(?<input>.*)|$)/s', $content, $match)) {
+                throw new \InvalidArgumentException(sprintf('File format is invalid.'));
```

Comment:
@keradus there are some more `sprintf` calls without any parameter. So maybe it is intended by you?